modules/history/src/views/full/conversation.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
```

Comment:
Try to keep the name of the file identical to its class name. Ex, here it should be called ConversationPicker.jsx